changelogs/8.17.asciidoc

```diff
@@ -25,5 +25,6 @@ https://github.com/elastic/apm-server/compare/v8.16.0\...v8.17.0[View commits]
 [float]
 ==== Added
 
+- Surface config parsing error under EA managed mode. {pull}14574[14574]
```

Comment:
```suggestion
- Surface config parsing error under EA managed mode by logging and marking EA input unit as failed {pull}14574[14574]
- Remove unnecessary hot reload under EA managed mode when apm tracing config is nil {pull}14865[14865]
```

Both of these should be moved to "Bug fixes" from "Added".

Reply:
updated